matching-notification-api/src/main/java/org/phenotips/matchingnotification/script/MatchingNotificationScriptService.java

```diff
@@ -193,7 +156,7 @@ public String setStatus(String ids, String status)
      */
     public boolean saveLocalMatches(List<PatientSimilarityView> similarityViews, String patientId)
```

Comment:
Not related to the pull request, but related to your changes: since you are reworking the APIs, we should also remove this method completely, matches should always be saved wherever they are found (on the back-end), velocity script code (and this method is only called form velocity) should not have to explicitly save matches

Reply:
Since we use REST in master no pint to rework it now.